packages/adapters/watcher/src/adapter.ts

```diff
@@ -8,30 +9,48 @@ import {
   AssetInfo,
   Report,
   PauseResponse,
-  VerifyResponse,
   WatcherAlertsConfig,
+  WatcherInvariantResponse,
 } from "./types";
 import { AssetVerifier } from "./verifiers";
 
 // Aggregation class for interfacing with all adapter functionality.
 export class WatcherAdapter {
   private readonly verifiers: Verifier[];
   private readonly pauser: Pauser;
+  private readonly context: VerifierContext;
+
+  // transactions sent to connext within the polling interval. cleared after
+  // each time `checkInvariants` is called
+  private readonly latestTransactions: Record<string, string[]> = {};
+
+  private push = false;
```

Comment:
what's this used for?

Reply:
oh we can delete this, looks like i forgot to do that